Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2.py

```diff
@@ -0,0 +1,209 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict, Any
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+TYPES = {
+    'threatTypes': ["MALWARE", "SOCIAL_ENGINEERING", "POTENTIALLY_HARMFUL_APPLICATION", "UNWANTED_SOFTWARE"],
+    'platformTypes': ["ANY_PLATFORM", "WINDOWS", "LINUX", "ALL_PLATFORMS", "OSX", "CHROME", "IOS", "ANDROID"]
+}
+
+
+class Client(BaseClient):
+    def url_request(self, body: Dict, full_url):
+        result = self._http_request(
+            method='POST',
+            json_data=body,
+            full_url=full_url)
+        return result
+
+
+def test_module(client: Client, client_body: Dict, full_url) -> str:
+    try:
+        # testing a known malicious URL to check if we get matches var
+        test_url = "http://testsafebrowsing.appspot.com/apiv4/ANY_PLATFORM/MALWARE/URL/"
+        body = build_request_body(client_body, test_url)
+        res = client.url_request(body, full_url)
+        if res.get('matches'):
+            message = 'ok'
+        else:
+            message = 'Error querying Google Safe Browsing. Expected matching respons, but received none'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return message
+
+
+def build_request_body(client_body, list_url):
```

Comment:
add type hints

Reply:
also, seems like a static class func to me

Reply:
done